presto-native-execution/presto_cpp/main/operators/BroadcastFactory.cpp

```diff
@@ -146,10 +146,16 @@ velox::BufferPtr BroadcastFileReader::next() {
 
   auto readFile = fileSystem_->openFileForRead(
       broadcastFileInfos_[readfileIndex_].filePath_);
-  auto buffer = AlignedBuffer::allocate<char>(readFile->size(), pool_, 0);
-  readFile->pread(0, readFile->size(), buffer->asMutable<char>());
   ++readfileIndex_;
   ++numFiles_;
+
+  // Empty broadcast file.
```

Comment:
Is this a bug fix? Would you extract it into a separate PR and add a test?

Reply:
Yes, raised a separate PR with test : https://github.com/prestodb/presto/pull/20053